frontend/src/components/notifications/notificationcard/NotificationCardController.jsx

```diff
@@ -1,24 +1,29 @@
+import moment from 'moment'
 import NotificationCardView from './NotificationCardView'
 
 /**
  * Creates a NotificationCard Component.
- * TO DO:
- * Implement any NotificationCardController logic if needed
+ * Due to the time variable not existing currently in the backend we are passing in a default time
+ * When the endpoint is implemented the code can be change as below and the eslint warning can be renabled
+ * time={moment(notification.time).fromNow()}
```

Comment:
can we add an issue about it to the issues list?

Reply:
yep will do